components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -152,6 +152,13 @@
 <config_Redi_horizontal_ramp_max>30e3</config_Redi_horizontal_ramp_max>
 <config_Redi_horizontal_ramp_max ocn_grid="WCAtl12to45E2r4">40e3</config_Redi_horizontal_ramp_max>
 
+<!-- submesoscale_eddy_parameterization -->
+<config_submesoscale_enable>.false.</config_submesoscale_enable>
```

Comment:
@jonbob is this okay to change?  Do you need to change this via the automated scipts?

Reply:
@vanroekel - that's fine to change. And I'll check the bld files and make sure they're up-to-date with Registry as part of my merging process